src/common/pf_lldp.c

```diff
@@ -359,17 +357,17 @@ static void pf_lldp_add_ieee_mac_phy (
  * Contains the IP address.
  *
  * @param p_ipaddr         In:    IP address
- * @param p_lldp_cfg       In:    LLDP configuration for this port
+ * @param p_port_cfg       In:    LLDP configuration for this port
  * @param p_buf            InOut: The buffer.
  * @param p_pos            InOut: The position in the buffer.
  */
 static void pf_lldp_add_management (
    const os_ipaddr_t * p_ipaddr,
-   const pnet_lldp_cfg_t * p_lldp_cfg,
+   const pnet_lldp_port_cfg_t * p_port_cfg,
```

Comment:
This function should not be port-specific. Th Management Address TLV is specific to the interface, not to any physical port connected to the interface.

Reply:
I removed this argument. We might later add some interface-related argument if necessary.
Done.